### PR TITLE
[10.0][FIX] full repository: openerp -> odoo

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
ff0e41e8ca186144bd3637cde18f9f74--comment-memes-funny-memes

facepalm-meme-picard-15

21t0xx

screen_shot_2015-06-17_at_1 50 46_pm_gcaoy0

tpsc4

### CODE REVIEW

File: ao_crm/migrations/9.0.2.0.0/post-migration.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright 2018 Eficent Business and IT Consulting Services S.L.
 # Copyright 2018 Tecnativa, S.L.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
-from openerp import api, SUPERUSER_ID
+from odoo import api, SUPERUSER_ID
```
Comment (LoisRForgeFlow) [inline ao_crm/migrations/9.0.2.0.0/post-migration.py:4]: post migration in version 9, so still openerp right

File: account_analytic_default_purchase/models/purchase.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 
-from openerp import api, fields, models
+from odoo import api, fields, models
```
Comment (LoisRForgeFlow) [inline account_analytic_default_purchase/models/purchase.py]: This module is version 8 :rofl:

File: ao_account_voucher/views/voucher_payment_receipt_view.xml [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<openerp>
-    <data>
+<odoo>
```
Comment (LoisRForgeFlow) [inline ao_account_voucher/views/voucher_payment_receipt_view.xml]: version 8, openerp still applies.

File: ao_crm_helpdesk_problem_track/models/qc_problem_track.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 # Copyright 2017 Eficent Business and IT Consulting Services S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import api, fields, models
+from odoo import api, fields, models
```
Comment (LoisRForgeFlow) [inline ao_crm_helpdesk_problem_track/models/qc_problem_track.py]: this module is version 9...

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
most of this is not needed as the modules you are fixing are not migrated to v10.

**Comment by aheficent** [comment]
testinprod

File: ao_crm/migrations/9.0.2.0.0/post-migration.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply ao_crm/migrations/9.0.2.0.0/post-migration.py:4]: the module is in v10

**Comment by MiquelRForgeFlow** [comment]
Well, why having modules of other versions in v10 branch? 🤔

Now I have revised all modules. Thus, I have maintained the changes in those modules that needed the change.

**Review by AaronHForgeFlow — COMMENTED** [review]
Why changing code of uninstallable modules?

**Pull request merged**